src/compiler/checker.ts

```diff
@@ -20803,6 +20802,21 @@ namespace ts {
             }
         }
 
+        function errorSubsequentVariableDeclarationMustHaveSameType(firstDeclaration: Declaration, firstType: Type, nextDeclaration: Declaration, subsequentType: Type): void {
+            const valueDeclarationSourceFile = getSourceFileOfNode(firstDeclaration);
+            const otherSpan = getErrorSpanForNode(valueDeclarationSourceFile, getNameOfDeclaration(firstDeclaration) || firstDeclaration);
+            const otherLocation = getLineAndCharacterOfPosition(valueDeclarationSourceFile, otherSpan.start);
+            const otherLocationDescription = `${valueDeclarationSourceFile.fileName} ${otherLocation.line}:${otherLocation.character}`;

```

Comment:
is the chrome debugger issue with template literals fixed? [It doesn't look like it](bugs.chromium.org/p/chromium/issues/detail?id=659515), so probably it's better not avoid template literals for now.